implementations/node/src/api/controllers/_dev/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as Router from 'koa-router';
+
```

Comment:
I will include the portions of this that remove the routing reliance, if the `index.html` assumption proves to be true.